java/example/calypso/android/omapi/src/main/kotlin/org/eclipse/keyple/example/calypso/android/omapi/activity/CoreExamplesActivity.kt

```diff
@@ -71,13 +71,15 @@ class CoreExamplesActivity : ExamplesActivity() {
                 readers.forEach {
                     addHeaderEvent("Starting explicitAidSelection with $poAid on Reader ${it.name}")
 
-                    val seSelector = SeSelector(SeCommonProtocols.PROTOCOL_ISO7816_3, null,
-                            SeSelector.AidSelector(SeSelector.AidSelector.IsoAid(poAid)))
+                    val seSelector = SeSelector.builder()
+                            .seProtocol(SeCommonProtocols.PROTOCOL_ISO7816_3)
+                            .aidSelector(SeSelector.AidSelector.builder().aidToSelect(poAid).build())
+                            .build()
                     val seRequest = SeRequest(seSelector, null)
 
                     addActionEvent("Sending SeRequest to select: $poAid")
                     try {
-                        val seResponse = (it as ProxyReader).transmit(seRequest)
+                        val seResponse = (it as ProxyReader).transmitSeRequest(seRequest, ChannelControl.KEEP_OPEN)
```

Comment:
To verify why this error wasn't detected in the previous build? May the CI script has to be updated to include this example.